hadoop-hdds/docs/content/feature/Snapshot.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Ozone Snapshots"
+weight: 1
+menu:
+   main:
+      parent: Features
+summary: Snapshots Support for Ozone.
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+## Introduction
+
+Snapshot feature for Apache Ozone object store allows users to take point-in-time consistent image of a given bucket. Snapshot feature enables various use cases including 
+ * backup/restore: Create hourly/daily/weekly/monthly snapshots for backup and restore.
+ * Archival and Compliance: Take snapshots for compliance purpose and archieve them as required.
+ * Replication and DR : Snapshots provide a frozen immutable image of the bucket on the source Ozone cluster. Snapshots can be used for replicating these immutable bucket images to remote DR site.
+ * Incremental Replication : Snapshots feature with efficent snapshot-diff can offer an effcient way to incremntally sync up remote copy of the bucket.
+
+## Snapshot APIs
+
+Snapshot feature is available through 'ozone fs' and 'ozone cli'. This feature can also be programattically accessed from native java Ozone object store client. The feature provides following functionalities :
+* Create Snapshot: Create an instantenous snapshot for a given bucket
+```shell
+ozone sh snapshot create [-hV] <bucket> [<snapshotName>]
+```
+* List Snapshots: List all snapshots of a given bucket
+```shell
+ozone sh snapshot list [-hV] <bucket>
+```
+* Delete snapshot: Delete a given snapshot for a given bucket
+```shell
+ozone sh snapshot delete [-hV] <bucket> <snapshotName>
+```
+* Snapshot Diff: Given two snapshots, list all the keys that are different betwenn the them.
+```shell
+ozone sh snapshot diff [-chV] [-p=<pageSize>] [-t=<continuation-token>] <bucket> <fromSnapshot> <toSnapshot>
+```shell
```

Comment:
Can remove the `shell` suffix for the closing quotes

Reply:
done!